Use tex-svg-full.js for MathJax3 (#2549)
JuliaDocs/Documenter.jl: CHANGELOG.md

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## Version [v1.X.Y] - YYYY-MM-DD

### Added

### Changed

> * The MathJax3 setup now uses `tex-svg-full.js` and additionally draws in `mhchem` by default ([#2549]). 
>   This new default allows to render chemical arrows as generated by Catalyst.jl without additional 
>   setup. It is also consistent with the setup used by Pluto.jl.

Reviewer: ```suggestion
* The MathJax3 setup now uses `tex-svg-full.js` and additionally draws in `mhchem` by default, allowing for chemistry symbols to be rendered (consistent with Pluto.jl). ([#2549])
```
